styleguide/gulpfile.js

```diff
@@ -581,9 +581,7 @@ gulp.task('validate', gulp.parallel('styles:validate', 'js:validate', 'axe'), ca
  *  Used to compile production ready SCSS and JS code.
  *
  */
-// todo make sure fractal:build is executed first
-// or it overrides the build dir
-gulp.task('compile', gulp.series('fractal:build', gulp.parallel('styles:build', 'styles:dist', 'styles:extract', 'sassdoc', 'iconfont', 'js:build', 'js:dist', 'images:minify'
+gulp.task('compile', gulp.series('fractal:build', gulp.parallel('styles:build', 'styles:dist', 'styles:inject', 'styles:extract', 'sassdoc', 'iconfont', 'js:build', 'js:dist', 'images:minify'
```

Comment:
Is er reden om hier de styles en js dist te doen? :) je doet ze nu dubbel (build) ;-)

Reply:
Mmm geen idee... @delrueba jij enig idee?

Reply:
Hoezo dubbel?
Build voert toch gewoon compile uit?